### DISCUSSION TITLE
"Greeting never received" when try to send email

### DISCUSSION THREAD

**Discussion by medzhidov** [postalserver/postal#3264] [Help with using Postal]
## Describe the bug

If I send about one message per second, there is no problem. However, when there are more than 10-20 messages in the queue and I send about 3-4 messages per second, I get this error. I don’t experience the same error when sending a lot more messages per second in Yandex Postbox, so the problem is somewhere in the postal server.

## Expected behaviour

No problem with connection)

## Screenshots

![CleanShot 2024-12-24 at 12 37 21](https://github.com/user-attachments/assets/2549b697-611f-4787-9971-44be3425bc8c)

## Additional information/context

We tried increasing the number of workers to 8, but it didn’t help us.

**Comment by willpower232** [comment]
Can you share a screenshot of one of the affected messages in the Postal UI? There would be extra information on the right hand side.

**Comment by medzhidov** [comment]
@willpower232 postal reject them before they come inside 😢 Now we switched to yandex postbox and want return to postal but we don't know what to do with this error(

**Reply by willpower232** [reply]
If you can't see any more information then you will need to use `postal logs smtp` to see what is actually happening and what error message is being returned to whatever you're using to send the emails.

**Reply by medzhidov** [reply]
> postal logs smtp

![CleanShot 2024-12-24 at 23 38 11](https://github.com/user-attachments/assets/fec17541-e773-4507-9563-7c4adbe8e67a)

**Reply by willpower232** [reply]
You run the command outside of the docker container. There should not be a need to execute any commands inside of the container.

**Reply by medzhidov** [reply]
> You run the command outside of the docker container. There should not be a need to execute any commands inside of the container.

Loot at the second line of screenshot please. This command executed inside docker container with smtp server inside it

**Reply by medzhidov** [reply]
Outside of container you can see the list of all containers in the server.
![CleanShot 2024-12-25 at 15 25 22](https://github.com/user-attachments/assets/8650d402-50dd-4e2c-bfe2-a6ee4d2739e4)

But inside container there is no any subcontainer:
![CleanShot 2024-12-25 at 15 26 25](https://github.com/user-attachments/assets/f75b5253-7fc2-4a74-8676-1bc391979576)

**Reply by medzhidov** [reply]
In docker logs there also no any errors related to this problem

**Reply by willpower232** [reply]
I'm presuming you didn't follow the normal install for Postal so you could try `docker compose logs smtp` if you can find where you're running it from.
